save-core/src/commonNonJsTest/kotlin/org/cqfn/save/core/integration/ClassicWarnTest.kt

```diff
@@ -0,0 +1,63 @@
+package org.cqfn.save.core.integration
+
+import org.cqfn.save.core.test.utils.runTest
+import org.cqfn.save.core.test.utils.runTestsWithDiktat
+
+import kotlin.test.Ignore
+import kotlin.test.Test
+
+// To run these tests locally on your Native platforms you would need to install curl for your OS:
+// On windows you'll also need to install msys2 and run pacman -S mingw-w64-x86_64-curl to have libcurl for ktor-client.
+// On ubuntu install libcurl4-openssl-dev for ktor client and libncurses5 for kotlin/native compiler.
+const val KTLINT_VERSION = "0.39.0"
+const val DIKTAT_VERSION = "1.0.0-rc.2"
```

Comment:
May be use variables from `TestUtilsCommon`?

Reply:
fixed